src/Tizen.NUI/src/public/Application/NUIApplication.cs

```diff
@@ -659,6 +659,17 @@ static public void Preload()
             IsPreload = true;
         }
 
+        /// <summary>
+        /// Set to true if BindableProperty is not used. 
+        /// This must be called immediately after the NUIApplication constructor is called.
+        /// The default value is false.        
+        /// </summary>
+        /// <remarks>
+        /// This must be called immediately after the NUIApplication constructor is called.
+        /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        static public bool DisableBindableProperty { get; set; } = false;
```

Comment:
```suggestion
        static public bool IsUsingXaml{ get; set; } = false;
```

Could you change the name of this property, pairwise with API11?